src/mudlet.cpp

```diff
@@ -126,9 +126,8 @@ bool TConsoleMonitor::eventFilter(QObject* obj, QEvent* event)
         mudlet::smDebugMode = false;
         mudlet::self()->refreshTabBar();
         return QObject::eventFilter(obj, event);
```

Comment:
Since this line executed no matter which branch of the `if` is taken - it can be moved OUT of the `if` altogether!